docs/changelog.md

```diff
@@ -24,6 +24,11 @@ Documentation
 -------------
 
 - Fixed 'Direct usage of document is deprecated' warning in the getting started guide. ([#1052](https://github.com/pyscript/pyscript/pull/1052))
+<<<<<<< Updated upstream
```

Comment:
I think this was a bad merge 😄

Reply:
I used GH GUI 🙄 to make this commit
Never trust the GUIs